packages/core/src/styles/typography/_legacy-typography.scss

```diff
@@ -0,0 +1,358 @@
+// Copyright (c) 2016-2020 VMware, Inc. All Rights Reserved.
+// This software is released under MIT license.
+// The full license information can be found in LICENSE in the root directory of this project.
+
+@import './../mixins/mixins';
+
+$typography-h0-color: hsl(0, 0%, 20%);
```

Comment:
The legacy typography I moved out of the tokens into a separate file since long term they will go away and it cleaned up the typography tokens quite a bit. If there is a reason they should still exist in the tokens I can add them back.